packages/block-library/src/social-links/editor.scss

```diff
@@ -11,8 +11,10 @@
 	transform: none;
 }
 
+// Align social links vertically to Page-Links text when using vertical Navigation block.
 .editor-styles-wrapper .wp-block-social-links {
 	padding: 0;
+	padding-left: 0.5em;
```

Comment:
What is this change for?

Reply:
That addresses the alignment in a vertical Nav block. When adding a vertical Nav block, the Social Icons don't seem to align vertically.

This is how it looks for me with padding-left: 0.5em
![withpadding](https://user-images.githubusercontent.com/62180768/106606895-15bd2700-657c-11eb-856c-d735f1fa5a71.png)

And this is how it looks without padding-left: 0.5em
![withoutpadding](https://user-images.githubusercontent.com/62180768/106606982-32f1f580-657c-11eb-9f68-1fdb3f6922ad.png)

Reply:
> That addresses the alignment in a vertical Nav block.

Does it not break alignment when the block is inside a post (as opposed to inside a nav block)?

Reply:
Thanks for pointing that out, you're correct, it does break the alignment when the block is inserted into a regular post - I should delete the `padding-left: 0.5em` line I added.

After looking further into this and testing, I believe the correct classname for this is `.wp-block-navigation ul` and `.wp-block-navigation ol` which is in packages/block-library/src/navigation/editor.scss.

Would you think it'd be a good idea to do the following? This should not affect the alignment when the block is inside a post but will help align the icons when they're inside a nav block.

```
// Undo default editor styles.
.editor-styles-wrapper .wp-block-navigation ul,
.editor-styles-wrapper .wp-block-navigation ol {
	margin-bottom: 0;
	margin-left: 0;
	padding-left: 0;
}

.wp-block-navigation ul, .wp-block-navigation ol {
        padding-left: 0.5em;
}
```

Reply:
Thanks for pointing that out, you're correct, it does break the alignment when the block is inserted into a regular post - I should delete the padding-left: 0.5em line I added.

After looking further into this and testing, it appears `.editor-styles-wrapper .wp-block-navigation ul` which is in packages/block-library/src/navigation/editor.scss is responsible for the padding here.

What do you think about changing the padding-left, like the following? This should not affect the alignment when the block is inside a post but will help align the icons when they're inside a nav block.

```
// Undo default editor styles.
.editor-styles-wrapper .wp-block-navigation ul,
.editor-styles-wrapper .wp-block-navigation ol {
	margin-bottom: 0;
	margin-left: 0;
	padding-left: 0.5em;
}
```

Reply:
I think it's a good instinct to improve this behavior, your screenshot clearly shows that this can be improved.

But I think it might be good to look at this in isolation, in a separate pull request. I imagine a dedicated PR that looks at the social links in a vertical, horizontal, standalone version, and massaging those margins could be productive. 

But this PR as is, with just the oblong shape fix, has value on its own. So it might be good to extract this change for now, so we can land the other fix! What do you think?

Reply:
I'm with you! Just pushed another update for this. Thanks again for your insight

Reply:
Perfect! Thank you :+1: